container/tabs.go

```diff
@@ -795,17 +815,20 @@ func (r *tabCloseButtonRenderer) Layout(size fyne.Size) {
 }
 
 func (r *tabCloseButtonRenderer) MinSize() fyne.Size {
-	return fyne.NewSquareSize(theme.IconInlineSize())
+	return fyne.NewSquareSize(r.button.Theme().Size(theme.SizeNameInlineIcon))
 }
 
 func (r *tabCloseButtonRenderer) Objects() []fyne.CanvasObject {
 	return r.objects
 }
 
 func (r *tabCloseButtonRenderer) Refresh() {
+	th := r.button.Theme()
```

Comment:
Having to do this everywhere seems cumbersome since only the theme, not the variant, can be overridden by ThemeOverrideContainer. So unless we plan on introducing variant overrides as well later, maybe it's worth thinking about adding a new theme helper that looks up the color given the theme and using the current app variant? Something to think about but not blocking this PR

Reply:
There is already `theme.ColorForWidget` which does just that, but it does not make use of the baseWidget cache we have for faster lookups internally.